playground/Mermaid.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useTransition, useEffect } from "react";
```

Comment:
This is responsible for rendering Mermaid Diagram so let's rename to `MermaidDiagram` to be more clear, rn it might get confused with the mermaid package.

Reply:
done in [96f6505](https://github.com/excalidraw/mermaid-to-excalidraw/pull/52/commits/96f6505e48da0c500ab64e63733b4e8d6a6b52e9)